app/src/main/java/com/example/news_pract/util/Constants.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.news_pract.util
+
+class Constants {
+    companion object{
+        const val API_KEY = "b59009fe30ee41aeb0cc57f1e5f309f8"
```

Comment:
you can make it a global constant. You don't need the companion object and the Constants class